jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +217,190 @@ def _get_project_status(project_statuses, status):
 
         self.api.user_stories.delete(story_id)
         return migrate_story.id
+
+    def list_all_issues(self, user_id=""):
+        """
+        List all issues for specific user if you didn't pass user_id will list all the issues
+        
+        Args:
+            user_id (int): id of the user.
+
+        Returns:
+            List: List of issues.
+        """
+        return self.api.issues.list(assigned_to=user_id)
+
+    def list_all_projects(self):
+        """
+        List all projects
+        
+        Returns:
+            List: List of projects.
```

Comment:
Here too, it applies for all return types and arguments docstring, it's better to specify the type as a list of what?